### PR TITLE
more logo

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by maelle** [comment]
@hadley 

logo on homepage https://6054bd0ff0eb574b884692d3--pkgdown-dev.netlify.app/
logo before TOC https://6054bd0ff0eb574b884692d3--pkgdown-dev.netlify.app/articles/pkgdown.html

File: inst/assets/BS4/pkgdown.css [hunk 1]
```diff
@@ -360,3 +361,27 @@ table {
...
+
+/* logo before TOC */
+#logo-toc {
+  display: grid;
+}
+
+/* logo homepage */
+#logo-homepage {
+  float: right;
+  width: 200px;
```
Comment (hadley) [inline inst/assets/BS4/pkgdown.css:373]:
```suggestion
  width: 180px;
```

I think this looks a little nicer

File: inst/templates/BS4/content-article.html [hunk 2]
```diff
@@ -55,9 +55,9 @@ <h4 data-toc-skip class="date">$date$</h4>
     </div>
 
     $if(toc)$
-    <div class="sticky-top">
+    <div class="sticky-top" id ="logo-toc">
```
Comment (hadley) [inline inst/templates/BS4/content-article.html:58]: I think this looks a little better below the contents. And can you add to the reference sidebar too? (Just to give me one more place to consider)

Comment (hadley) [reply inst/templates/BS4/content-article.html:58]: With maybe `margin-top: 3em`

**Pull request closed without merging**